cognite_toolkit/_builtin_modules/inrobot/cdf_inrobot_location/module.toml

```
@@ -1,5 +1,6 @@
 [module]
 title = "InRobot Location"
+default_selected = true
```

Suggest to call this 
```suggestion
is_selected_by_default = true
```